### PR TITLE
upd pltm bfn

### PR DESCRIPTION
**Pull request by vboykox**
Signed-off-by: Volodymyr Boyko <volodymyrx.boiko@intel.com>

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

**Comment by vboykox** [comment]
please review but do not merge yet

**Comment by vboykox** [comment]
@akokhan  can be merged

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py [hunk 1]
```diff
@@ -307,40 +315,58 @@ def get_transceiver_info(self):
     def get_transceiver_bulk_status(self):
         status = dict()
 
+        chan_count = self.__chan_count_get()
+
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py]: Please remove empty line

File: platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py [hunk 2]
```diff
@@ -307,40 +315,58 @@ def get_transceiver_info(self):
...
+        if chan_count is None:
+            return None
+
         rx_los = self.get_rx_los()
         if rx_los is not None:
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py]: Can we made it one line here and below?
```
rx_los = self.get_rx_los()
status['rx_los'] = any(rx_los) if rx_los is not None else 'N/A'
```

Comment (vboykox) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py]: looks more readable to you?

Comment (vboykox) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py]: yeah

Comment (vboykox) [reply platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/sfp.py]: will change

**Review by akokhan — APPROVED** [review]

**Pull request merged**